tests/_support/AcceptanceTester.php

```diff
@@ -64,4 +64,21 @@ public function bootstrapWith( callable $function, string $file = null, bool $ro
 
 		return $rollback_callback;
 	}
+
+	/**
+	 * Run ElasticPress indexing command.
+	 *
+	 * WPBrowser DB management commands to not trigger WordPress actions, and hence Elastic does not keep track
+	 * of content updates and index them. This command makes sure Elastic index is updated to reflect the changes.
+	 *
+	 * @param string $options Additional arguments to append to the indexing command.
+	 * @return void
+	 */
+	public function reindexElastic( string $options = '' ) {
+		// phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.system_calls_exec
+		exec( sprintf(
+			'WPBROWSER_HOST_REQUEST=1 wp elasticpress index --network-wide --setup --yes %s',
```

Comment:
I'm in 2 minds about whether `--setup` is needed but better to be safe. There's never that much data!

Reply:
Yup, had the same concern.